framework/widgets/ActiveField.php

```diff
@@ -800,4 +837,22 @@ protected function getInputId()
     {
         return $this->_inputId ?: Html::getInputId($this->model, $this->attribute);
     }
+
+    /**
+     * adds aria attributes to the input options
+     * @param $options array input options
+     */
+    protected function addAriaAttributes(&$options)
+    {
+        if ($this->addAriaAttributes) {
+
+            $options['aria-required'] =
+                (!isset($options['aria-required']) && $this->model->isAttributeRequired($this->attribute))
+                ? 'true' : 'false';
```

Comment:
This line is redundant. PHP boolean will be converted to `true` or `false` automatically